doc/manual/command-ref/conf-file.xml

```diff
@@ -302,6 +302,20 @@ flag, e.g. <literal>--option gc-keep-outputs false</literal>.</para>
   </varlistentry>
 
 
+  <varlistentry><term><literal>build-hook-use-substitutes</literal></term>
```

Comment:
I don't like this name since "build hook" is just an implementation detail used by the distributed build mechanism.